content/discover/who-uses.json

```diff
@@ -88,6 +88,11 @@
       "url": "http://architectnow.net",
       "width": "150px"
     },
+    {
+      "logo": "/assets/logo/radity.png",
+      "url": "https://radity.com",
+      "width": "150px"
+    },
```

Comment:
Can you please add it to the end of the list?

Reply:
@kamilmysliwiec thank you for your feedback. I got it